src/objects/imageButton.js

```diff
@@ -24,15 +18,4 @@ const ImageButton = class
 	{
 		image(this.img, this.x, this.y);
 	}
-
-	init()
```

Comment:
You asked how to simplify files like cheatSheet.js. Init and destroy were meant to abstract out the process of adding and removing callbacks to and from the mouseClicked event. Maybe that code could move that code here.

Reply:
Yeah that might be a good idea, I put that list of image buttons inside the scene file as I felt it made more sense there - that list isn't really owned by this image button class, it's more something that's specific to each scene. Definitely worth looking closer at that though, for now I just wanted to focus on the modules refactor here and get everything working again.